src/cursor/cur_std.c

```diff
@@ -1194,12 +1202,17 @@ __wt_cursor_bound(WT_CURSOR *cursor, const char *config)
                   session, CUR2BT(cursor)->collator, &key, &cursor->lower_bound, &exact));
                 if (exact < 0)
                     WT_ERR_MSG(session, EINVAL, "The provided cursor bounds are overlapping");
+                /* If lower bound and upper bound is equal, both inclusive flags must be set. */
```

Comment:
Reword: "If the lower bound and upper bound are equal, both inclusive flags must be specified. ".